src/globals.js

```diff
@@ -5,6 +5,9 @@ import { createElement } from '@wordpress/element';
 import jsdom from 'jsdom-jscore';
 import jsdomLevel1Core from 'jsdom-jscore/lib/jsdom/level1/core';
 
+// Import for side-effects: Patches for jsdom-jscore.
+import './jsdom-patches';
```

Comment:
Hmm, this seems rather important since we're touching how `jsdom-jscore` is working. I'm sure we need the patches so, can you add some comments in the code on what do we need the patches for? Thanks!

Reply:
Good idea, I'll add some more comments to this to clarify.

Reply:
Thanks @mkevins ! Can we expand on the reason(s) we replace each of the function prototypes? That insight will help anyone reading the code to understand why do we even replace these function.

Reply:
I've added PR #826 to address this.